benchmarks/randomized_benchmarking.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2022 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import List
+import functools
+import numpy as np
+import cirq
+import cirq.experiments.qubit_characterizations as ceqc
+
+
+def dot(*args: np.ndarray) -> np.ndarray:
+    return functools.reduce(np.dot, args)
+
+
+class SingleQubitRandomizedBenchmarking:
+    params = [[1, 10, 50, 100, 250, 500, 1000], [100], [20]]
```

Comment:
Is this unsafe to have a mutable array as the default?

Reply:
Yes, the value of params is read by asv to run the class multiple times for different parameter configurations. This is the recommended way of writing parameterized benchmarks. 

See https://asv.readthedocs.io/en/stable/writing_benchmarks.html#parameterized-benchmarks for more details.